assets/wire/components/fields/ExpiryDateLabel.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {gettext, formatDate} from 'utils';
+
+export function ExpiryDateLabel ({item, filterGroupLabels}: any) {
```

Comment:
@dzonidoo Please add types as per the PR checklist. This includes adding `expiry` to the `IArticle` (https://github.com/superdesk/newsroom-core/blob/develop/assets/interfaces/content.ts#L13), and avoiding use of `any` and `PropType` and instead use the appropriate `IProps` interface to be used for this component